truenas_installer/__main__.py

```
@@ -20,9 +21,15 @@ def main():
     with open("/etc/version") as f:
         version = f.read().strip()
 
+    try:
```

Let's be extra safe here. I'd rather just catch all exceptions and pass.

```
vendor = "TrueNAS"
try:
    with open(...
        vendor = ...
except Exception:
    pass